db/seeds.rb

```diff
@@ -30,7 +30,7 @@
                          price: 50.0,
                          image_path: "#")
 
-admin = FactoryGirl.create(:user, role: 1, username: "admin", password: "password")
+admin = User.create(first_name: "Maria", last_name: "Smith", role: 1, username: "admin", password: "password")
```

Comment:
FactoryGirl might have worked if we required it in this file, but since we're not using it in any of our other creations, I'm not too concerned about it.